hat/conf/logback.xml

```diff
@@ -81,6 +81,15 @@
     <logger name="org.hatdex.hat.api.service.monitoring" level="INFO" additivity="false">
         <appender-ref ref="ASYNCCONSOLE"/>
     </logger>
+    <logger name="org.hatdex.hat.she.service.monitoring.HatDataEventsProcessor" level="DEBUG" additivity="false">
```

Comment:
Please set the appropriate log level, it shouldn't be DEBUG when merging into production branch.

Reply:
DEBUG level was intentional. It would be too much to log into AWS with INFO level.

This is also the same setting as SHE functions a few lines down.

No action taken

Reply:
If the logs have high enough priority to be used in production environment, they should be upgraded to INFO. If they are actually used for debugging they should not end up in the master branch. But I see how the current deployment process does not allow for easy management between test/production.